geode-core/src/distributedTest/java/org/apache/geode/cache/ConnectionPoolDUnitTest.java

```diff
@@ -1318,7 +1299,7 @@ public void test010LifetimeExpireOnPoolCnx() throws CacheException {
   protected static volatile int baselineLifetimeConnect;
   protected static volatile int baselineLifetimeDisconnect;
 
-  private void basicTestLifetimeExpire(final boolean threadLocal) throws CacheException {
+  private void basicTestLifetimeExpire() throws CacheException {
```

Comment:
Is this method still called? I think it should be. The old test method that did it was "test010LifetimeExpireOnPoolCnx" but that was deleted. Seems like you could just make this on public, rename it, and add the Test annotation

Reply:
Good catch. I think that is exactly what I intended to do and missed it.